sources/packages/web/src/types/contracts/EducationProgram.ts

```diff
@@ -50,12 +50,12 @@ export class EntranceRequirements {
 }
 
 export interface ProgramOfferingHeader {
-  institutionId: number;
-  institutionName: string;
-  submittedDate: string;
+  institutionId?: number;
```

Comment:
I believe `institutionId` is not used anywhere.

Reply:
I believe it is used for navigation when the link is clicked from header.

Reply:
Yes, used for navigation when the link is clicked from header.

Reply:
If is it used it needs to be provided. I did not find any reference to it and some methods are missing, for instance, the below ones.

https://github.com/bcgov/SIMS/blob/2c3e6c3138116ede3193d72b55e660e24d5a9a88/sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequest.vue#L116-L123

https://github.com/bcgov/SIMS/blob/92c8b1e8f1e8c120c9fe2333c49d56236a13eb44/sources/packages/web/src/views/aest/institution/ViewOfferingChangeRequestComplete.vue#L73-L80

Reply:
This is a previously existing bug and will require a change in multiple files. Should I do it as a part of this PR?

Reply:
I added the comment because your name appeared in the recent changes.
It seems that the `institutionId` was changed to optional when it needed to be mandatory, and the same for `institutionName`.

![image](https://github.com/bcgov/SIMS/assets/61259237/62754fab-7e9f-40c6-914e-1b4cdfe440d3)

Also tracing back the data being populated it also indicated your name.

![image](https://github.com/bcgov/SIMS/assets/61259237/1f3110b5-52c2-4716-a199-c94e67e5ed08)

If this is associated with an issue that we already have a ticket for, please just revert the two mandatory properties that were changed to optional.